docs/standards.md

```diff
@@ -0,0 +1,167 @@
+## SG2 Standards Checklist
+
+Since SG2 is a living design library that is home to reusable patterns, it's important for all contributing developers to attempt to create patterns the same way. 
+
+Any new work contributed into SG2 must meet the following standards:
+
+### PatternLab Organization
+
+#### How patterns are organized:
+- Patterns should be listed alphabetically in the pattern list. No numbers (e.g. `atoms/forms/text-input` instead of `00-atoms/10-forms/05-text`). 
+- Talk about where SCSS and JS files should go (in a separate .scss directory?), as well as Twig and .md files (no subdirectories?). We may still need to make a decision about this.
+
+##### Atoms
+- An atom should be the basic HTML elements.
+- They can sometimes be more than one element (e.g. an unordered list).
+
+##### Molecules
+- A molecule should include most of the markup and pull in atoms as necessary.
+- Not every element in a molecule needs to be an atom.
+
+##### Organisms
+- Organisms should be very little markup, it should be mostly wrappers around molecules or atoms.
+
+##### Templates
+- Layouts should be used for templates but not molecules
+
+##### Pages
+Add a description here.
+
+##### Base
+Add a description of the "base" patterns here. Do they always get reused, or no?
+
+Contains:
+- "Colors" atom showing variables and base color palette.
+
+##### Meta
+Add a description of the "meta" patterns here. This section still needs to be added to SG2.
```

Comment:
What do we want to include in the Meta section? What is the use case for this?

Reply:
Good call - it looks like it's probably not needed, taking another look at Mayflower. They have a specific use case for it and we don't